modules/building/pages/prefetching-dependencies.adoc

```diff
@@ -256,12 +294,18 @@ spec:
 
 NOTE: Konflux also supports prefetching RPM content which requires a Red Hat subscription. For more information see xref:./activation-keys-subscription.adoc#hermetic-network-isolated-builds[Using Red Hat activation keys to access subscription content].
 
+=== [[rpm-walkthrough]]`rpm` walkthrough
+
+Refer to the link:https://hermetoproject.github.io/hermeto/rpm/#example[rpm walkthrough] for a detailed example of using rpm with Hermeto.
```

Comment:
RPM is tricky, because the example is very simplistic, but often times folks fight with the [extra JSON options](https://hermetoproject.github.io/hermeto/rpm/#specifying-packages-to-process) and the fact that they need a lockfile in the first place (this is documented though). So, I'm wondering how we could also steer the focus to the JSON options doc.

Reply:
There's a whole other set of RPM stuff in "modules/building/partials/", e.g. `konflux-ci-docs/modules/building/partials/konflux-example-prefetch-rpm-*.adoc`, *but I don't understand how those are linked into the actual docs*.